msocchistory.yaml

```diff
@@ -0,0 +1,112 @@
+entityDefinitions:
+  charge:
+    fqn: "ol.offense"
+    entitySetName: "MSOCharge"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "CHARGE_PK"
```

Comment:
Same q about the charge pk as above.

Reply:
See comment above - I'll commit my changes so far and continue to work on the integration for now with charge_pk as the key for charge. I can change that if you think it should be deduplicated on something else and I could add something to the wiki since I was unable to find guidance for how they should be deduplicated on the Charge and Pleas

Reply:
see comment above

Reply:
see comment above